examples/training/classification/densenet/imagenet/basic_training.py

```diff
@@ -0,0 +1,234 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import sys
+
+import tensorflow as tf
+from absl import flags
+from keras.callbacks import BackupAndRestore
+from keras.callbacks import EarlyStopping
+from keras.callbacks import ModelCheckpoint
+from keras.callbacks import TensorBoard
+from keras.layers import Resizing
+from keras.losses import CategoricalCrossentropy
+from keras.optimizers import Adam
```

Comment:
@ianjjohnson I think it would be nice to import keras from tensorflow. I've encountered many practitioners who just mix old and new keras in the same environment and get import errors.

Reply:
Thanks for calling this out -- done.